rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/60_synonym_rule_get.yml

```diff
@@ -0,0 +1,41 @@
+setup:
+  - skip:
+      version: " - 8.8.99"
```

Comment:
We need to put 8.9.99 for "Introduced in 8.10.0" ( at least TODO if this version is not yet available)